packages/communication-react/review/communication-react.api.md

```diff
@@ -426,6 +426,14 @@ export interface CameraButtonStrings {
     onLabel: string;
 }
 
+// @public (undocumented)
+export const CHAT_UI_IDS: {
```

Comment:
nit (though realize this is draft but leaving this here anyway): these look like they should be excluded from the public api

Reply:
Counter-point: integration tests should be black-box tests -- if they need IDs for components inside the composite, some way to get at those IDs should be part of the public API.

Making integration tests grey-box tests inevitably leads to some business logic in the system under test that is test-specific.

Reply:
+1 to what @prprabhu-ms said. By not exposing these constants in the public API we end up doing absolute imports and break the ability of this test to be run on a black box composite...

Reply:
I have mixed feelings about this. Would love to know what others think

Reply:
I think keep them public, I've been swayed 😄 easier to keep them public and have the azure review board remove them than the other way around

Reply:
I kind of like the idea that we have these ids here so if someone else wants to do end to end testing with UI components they have a bunch of hooks to have good end to end tests?

Reply:
Is acs the official acronym for our service?

Reply:
With the ID injection approach (OK to do a follow up PR), ID collision is not an issue.